notebooks/ImvizExample.ipynb

```diff
@@ -92,69 +107,65 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "acs_47tuc_1 = download_file('https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jbqf03gjq_flc.fits', cache=True)\n",
-    "acs_47tuc_2 = download_file('https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HST/product/jbqf03h1q_flc.fits', cache=True)"
+    "data_dir = tempfile.gettempdir()\n",
+    "#data_dir = \"/User/username/Data/\"\n",
+    "\n",
+    "files = ['jw02727-o002_t062_nircam_clear-f090w_i2d.fits',\n",
+    "         #'jw02727-o002_t062_nircam_clear-f150w_i2d.fits',\n",
+    "         #'jw02727-o002_t062_nircam_clear-f200w_i2d.fits',\n",
+    "         'jw02727-o002_t062_nircam_clear-f277w_i2d.fits',\n",
+    "         #'jw02727-o002_t062_nircam_clear-f356w_i2d.fits',\n",
+    "         #'jw02727-o002_t062_nircam_clear-f444w_i2d.fits'\n",
+    "        ]\n",
+    "\n",
+    "for fn in files:\n",
+    "    uri = f\"mast:JWST/product/{fn}\"\n",
+    "    result = Observations.download_file(uri, local_path=f'{data_dir}/{fn}')\n",
+    "    with warnings.catch_warnings():\n",
+    "        warnings.simplefilter('ignore')\n",
+    "        imviz.load_data(f'{data_dir}/{fn}')"
```

Comment:
By default, when Imviz loads the two images, the view smaller because the first image loaded is a "smaller" image. This goes away once you link by WCS, but I'm wondering whether we want to switch the two datasets so that at least the "large" image is on top (and takes up the full screen)? That being said, the "smaller" one is prettier IMHO

<img width="995" alt="Screen Shot 2022-10-12 at 4 12 28 PM" src="https://user-images.githubusercontent.com/25206008/195438520-e84ce7d9-5848-4c45-ae73-2c2d77ff107f.png">

Reply:
Read the next cell 😆

Reply:
I retract my statement... lol my brain is done with today :P

Reply:
All good 😁 . I wanted exactly this to demonstrate the differences in the linking options.